src/components/AreaOfResponsibiltySelector.tsx

```diff
@@ -0,0 +1,101 @@
+import { TicketSearchParam } from '../api/searchApi';
+import { RootState } from '../redux/store';
+import { dataTestId } from '../utils/dataTestIds';
+import { getLanguageString } from '../utils/translation-helpers';
+
+export const AreaOfResponsibilitySelector = () => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const nvaUsername = user?.nvaUsername ?? '';
+
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const institutionUserQuery = useQuery({
+    enabled: !!nvaUsername,
+    queryKey: ['user', nvaUsername],
+    queryFn: () => fetchUser(nvaUsername),
+    meta: { errorMessage: t('feedback.error.get_person') },
+  });
+  const areasOfResponsibilityIds = institutionUserQuery.data?.viewingScope?.includedUnits ?? [];
+
+  const organizationQuery = useQuery({
+    enabled: !!areasOfResponsibilityIds,
+    queryKey: ['organizations', areasOfResponsibilityIds],
+    queryFn: async () => {
+      const areaPromises = areasOfResponsibilityIds.map(async (orgId) => {
+        return await fetchOrganization(orgId);
+      });
+      return await Promise.all(areaPromises);
+    },
+    meta: { errorMessage: t('feedback.error.get_institution') },
+  });
+  const organizationOptions = organizationQuery.data ?? [];
+
+  const selectedAreaIdsFromUrl = searchParams.get(TicketSearchParam.ViewingScope)?.split(',');
+  const selectedOrganizations = organizationOptions.filter((org) => selectedAreaIdsFromUrl?.includes(org.id));
+
+  return (
+    <Autocomplete
+      multiple
+      autoHighlight
+      options={organizationOptions}
+      value={selectedOrganizations}
+      disableCloseOnSelect
+      disabled={organizationQuery.isLoading}
+      loading={organizationQuery.isLoading}
+      getOptionLabel={(option) => getLanguageString(option.labels)}
+      renderTags={(values, getTagProps) => (
+        <Chip
+          sx={{ py: '0.1rem' }}
+          label={t('common.chosen', { count: values.length })}
+          color="primary"
+          size="small"
+          variant="filled"
+          data-testid={dataTestId.registrationWizard.resourceType.journalChip}
+          onDelete={() => {
+            searchParams.delete(TicketSearchParam.ViewingScope);
+            history.push({ search: searchParams.toString() });
+          }}
+        />
+      )}
+      onChange={(_, values) => {
+        if (values.length) {
+          searchParams.set(TicketSearchParam.ViewingScope, values.map((org) => org.id).join(','));
+        } else {
+          searchParams.delete(TicketSearchParam.ViewingScope);
+        }
+        history.push({ search: searchParams.toString() });
+      }}
+      renderOption={(props, option, { selected }) => (
+        <li {...props} key={option.id}>
+          <Checkbox
+            icon={<CheckBoxOutlineBlankIcon fontSize="small" />}
+            checkedIcon={<CheckBoxIcon fontSize="small" />}
```

Comment:
ser ut som dette er gjort bare for å få små ikoner?
```suggestion
            size="small"
```

Reply:
Her er det nok min copy-paste som har tatt litt hardt i ja 😄